wagtail-{{ cookiecutter.project_name_kebab }}/pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "wagtail-{{ cookiecutter.project_name_kebab }}"
+authors = [{name = "{{ cookiecutter.full_name }}", email = "{{ cookiecutter.email }}"}]
+description = "{{ cookiecutter.project_short_description }}"
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    {% if cookiecutter.open_source_license == 'MIT license' -%}
+    "License :: OSI Approved :: MIT License",
+    {%- elif cookiecutter.open_source_license == 'BSD license' -%}
+    "License :: OSI Approved :: BSD License",
+    {%- elif cookiecutter.open_source_license == 'ISC license' -%}
+    "License :: OSI Approved :: ISC License (ISCL)",
+    {%- elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}
+    License :: OSI Approved :: Apache Software License",
+    {%- elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    {%- elif cookiecutter.open_source_license == 'Not open source' -%}
+    "License :: Other/Proprietary License",
+    {% endif %}
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Framework :: Django",
+    "Framework :: Django :: 3.0",
+    "Framework :: Django :: 3.1",
+    "Framework :: Django :: 3.2",
+    "Framework :: Wagtail",
+    "Framework :: Wagtail :: 2",
+]
+dynamic = ["version"]
+dependencies = [
+    "Django>=3.0,<4.0",
+    "Wagtail>=2.14,<2.16"
+]
+[project.optional-dependencies]
+testing = [
+    "dj-database-url==0.5.0",
+    "freezegun==0.3.15"
```

Comment:
may as well do 1.2.2 or >=1.2,<1.3

Reply:
I've pinned both in 1f76c2b . Dev/test dependencies can be hard-pinned to a version. With dev dependencies, we're trying to optimise for reproducibility unlike the main dependencies where we're trying to optimise for compatibility.

Tbh, the intent of this PR is to migrate from setuptools to flit. This change is unrelated (but doing this is 2 lines of scope creep, so lets just get it done), but looking at this has made me question if we should bundle freezegun at all. I'm going to move the discussion to another issue.

Reply:
moved the discussion to https://github.com/wagtail/cookiecutter-wagtail-package/issues/36 It has no bearing on whether we are using setuptools or flit as a build backend.